app/uk/gov/hmrc/eusubsidycompliancefrontend/services/BusinessEntityJourney.scala

```diff
@@ -53,6 +54,8 @@ object BusinessEntityJourney {
           empty.eori.copy(value = Some(eori)),
           empty.contact.copy(value = cd)
         )
+      // TODO - what is the correct behaviour here?
+      case None => BusinessEntityJourney()
```

Comment:
We should probably think about this. I think we should redirect at first thought. But we'll need to put this in a lot of places.

Reply:
OK - I'll leave the TODO here as a reminder.